src/components/study-pane.js

```diff
@@ -144,6 +147,8 @@ const StudyPane = (props) => {
         (state) => state.filteredNominalVoltages
     );
 
+    const [displayedSubstationId, setDisplayedSubstationId] = useState(null);
+
```

Comment:
Are we obliged to manage 2 state variables : displayedSubstationId & displayedVoltageLevelId
They are exclusive( only one and only one SLD)